php/class-terminus-command.php

```diff
@@ -180,11 +180,11 @@ protected function waitOnWorkflow( $object_name, $object_id, $workflow_id ) {
     $desc = $workflow['data']->active_description;
     $type = $workflow['data']->type;
     $tries = 0;
-    while( $result !== 'succeeded' AND $tries < 100) {
+    while(($result !== 'succeeded') AND ($result !== 'aborted') AND ($tries < 100)) {
```

Comment:
I think we can just check for the absence of a result, rather than enumerating all the possible results (which might change in the future).